api/src/main/resources/defaultAtomfeedConfig.json

```diff
@@ -38,7 +38,7 @@
       "category": "visit",
       "linkTemplates": {
         "rest": "/ws/rest/v1/visit/{uuid}?v=full",
-        "fhir": "/ws/fhir/Visit/{uuid}"
+        "fhir": "/ws/fhir/Encounter/{uuid}"
```

Comment:
Why did you change visit to encounter? A visit contains encounters, so it could work for a case when a visit has only one encounter. Is it a mistake or a part of a new solution?

Reply:
I guess it's a part of the new solution. See https://issues.openmrs.org/browse/SYNCT-210 for more details. It wasn't part of SYNCT-191 issue, but I guess @areklalo already did as it doesn't require a lot of changes.

Reply:
Regarding to https://wiki.openmrs.org/display/projects/FHIR+Resource+List+for+Sync
The OpenMRS Visit is mapped to the FHIR Encounter. It was mistake in the configuration.